podcasts/Sharing/ActivityItemSourceItem.swift

```diff
@@ -0,0 +1,20 @@
+class ActivityItemSourceItem: NSObject, UIActivityItemSource {
+    let item: Any
+    let disallowedActivityTypes: [UIActivity.ActivityType]?
```

Comment:
Nit: could this be a Set instead? If we don't care about the order, it could be better when checking the content